tests/garage/tf/regressors/test_categorical_mlp_regressor_with_model.py

```diff
@@ -0,0 +1,199 @@
+import pickle
+
+from nose2.tools.params import params
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.regressors import CategoricalMLPRegressorWithModel
+from tests.fixtures import TfGraphTestCase
+
+
+def get_labels(input):
+    label = [0, 0]
+    label[0 if np.sin(input) <= 0 else 1] = 1
+    return label
+
+
+class TestCategoricalMLPRegressorWithModel(TfGraphTestCase):
+    def test_fit_normalized(self):
```

Comment:
please also test the fit methods with a range of reasonable configuration values (catch 0-vs-1, 1-vs-2, 2-vs-many errors, etc.)